tests/unit/ipam_test.py

```diff
@@ -0,0 +1,46 @@
+from netaddr import IPNetwork, IPAddress
+from nose.tools import assert_equal, assert_true
+from node.adapter.ipam import SequentialAssignment, IPAMClient
+
+pool = IPNetwork("192.168.0.0/16")
+client = IPAMClient()
```

Comment:
you haven't mocked out etcd, so this will modify live data, and will fail if etcd isn't running.